plugins/modules/copr.py

```diff
@@ -255,6 +265,10 @@ def _enable_repo(self, repo_filename_path, repo_content=None):
         """
         if not repo_content:
             repo_content = self._download_repo_info()
+        if self.ansible_module.params.get("includepkgs"):
```

Comment:
There's no need to use `get()` here:
```suggestion
        if self.ansible_module.params["includepkgs"]:
```

Reply:
yup will update this , checked it was not necessary